firebase-installations/src/main/java/com/google/firebase/installations/FirebaseInstallations.java

```diff
@@ -109,6 +109,11 @@ public Thread newThread(Runnable r) {
           + "with Firebase server APIs: It identifies your application with Firebase."
           + "Please refer to https://firebase.google.com/support/privacy/init-options.";
 
+  private static final String AUTH_ERROR_MSG =
+      "Installation ID could not be validated with the Firebase servers (maybe deleted). "
```

Comment:
"The Installation ID [...]" or "Your Installation ID [...]"
I'm not sure, maybe without is best. :)

"[...] (maybe it was deleted)."

"[...] new Installation ID and auth token."